doc/Home.md

```diff
@@ -0,0 +1,36 @@
+# How the angular-component-extractor works
+
+When the extension is triggered, then it first reads the currently selected content of the html file.
+After ensuring that all requirements for a successful execution are met (see `Startup Requirements`), the following steps are processed:
+
+1. Extract interpolation
+2. Create one `update change` for each interpolation which adds a `@Input` to the new component ts
+3. Create one `replace change` for the new component html, which inserts the selection
+4. Provide the replacement for the selected html
+5. Apply changes
+
+![General workflow of the extension](./images/general-flow.png)
```

Comment:
As the image is hard to see if someone is using the Dark Mode of GitHub, it'd be better if you could replace the transparent background color with a white background:

![image](https://user-images.githubusercontent.com/12248527/125191262-c9b89180-e241-11eb-8a71-67963890b808.png)

Reply:
Fixed it.